builder/googlecompute/driver_mock.go

```diff
@@ -117,17 +119,23 @@ func (d *DriverMock) CreateImage(name, description, family, zone, disk string, i
 	if d.CreateImageResultSizeGb == 0 {
 		d.CreateImageResultSizeGb = 10
 	}
-
+	imageFeatures := make([]*compute.GuestOsFeature, 0, len(image_features))
+	for _, v := range image_features {
+		imageFeatures = append(imageFeatures, &compute.GuestOsFeature{
+			Type: v,
+		})
+	}
 	resultCh := d.CreateImageResultCh
 	if resultCh == nil {
 		ch := make(chan *Image, 1)
 		ch <- &Image{
-			Labels:    d.CreateImageLabels,
-			Licenses:  d.CreateImageLicenses,
-			Name:      name,
-			ProjectId: d.CreateImageResultProjectId,
-			SelfLink:  d.CreateImageResultSelfLink,
-			SizeGb:    d.CreateImageResultSizeGb,
+			GuestOsFeatures: imageFeatures,
+			Labels:          d.CreateImageLabels,
+			Licenses:        d.CreateImageLicenses,
+			Name:            name,
+			ProjectId:       d.CreateImageResultProjectId,
+			SelfLink:        d.CreateImageResultSelfLink,
+			SizeGb:          d.CreateImageResultSizeGb,
```

Comment:
Nice idea to keep these ordered by letter, I'll add that one to my personal coding style :D